presto-docs/src/main/sphinx/release/release-0.116.rst

```diff
@@ -2,6 +2,24 @@
 Release 0.116
 =============
 
+Cast between JSON and VARCHAR
+-----------------------------
+
+Casts of both directions between JSON and VARCHAR have been removed. New
```

Comment:
I would word this as:

Casts between JSON and VARCHAR have been removed. 
In a future release, we intend to add casts between JSON and BOOLEAN, BIGINT,
DOUBLE, and VARCHAR. The new semantics of the cast between JSON and VARCHAR will be
consistent with the semantics of the cast between JSON and BOOLEAN, BIGINT, and DOUBLE.
That is `CAST(JSON '"test"' AS VARCHAR)` will be equivalent to `'test'` and `CAST(JSON '[1]' AS VARCHAR)` 
will result in an error.
To get the semantics of the current casts, use:

* `JSON_PARSE(x)` instead of `CAST(x as JSON)`
* `JSON_FORMAT(x)` instead of `CAST(x as VARCHAR)`